spec/user_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'spec_helper'
```

Comment:
There are two helpers for rspec - one of them is `rails_helper` and the other is `spec_helper`
Difference is in that first one loads all the Rails environment and second does not. As we write test for rails model we need to use `rails_helper`